### PR TITLE
Updating .gitignore

### PR DESCRIPTION
**Pull request by AlleyCatRacer**

### CODE REVIEW

**Review by SkyKalazar — APPROVED** [review]
Seems good, lines were written, green is a nice colour

File: .gitignore [hunk 1]
```diff
@@ -31,3 +34,332 @@ google-services.json
 
 # Android Profiling
 *.hprof
+
+# Kotlin project
+.idea/shelf
```
Comment (AlleyCatRacer) [inline .gitignore:39]: Is this really a shelf?

**Comment by AlleyCatRacer** [comment]
Look at shelf comment

**Comment by SkyKalazar** [comment]
Hola, que tal?!

**Review by AlleyCatRacer — COMMENTED** [review]
Yes it is

File: .gitignore (hunk 1 above)
Comment (SkyKalazar) [inline .gitignore:39]: This is a line of code number 39

File: .gitignore [hunk 2]
```diff
@@ -31,3 +34,332 @@ google-services.json
...
+/dependencies/repo
+/android.tests.dependencies
+/dependencies/android.tests.dependencies
+/dist
+/local
+/gh-pages
+/ideaSDK
+/clionSDK
+/android-studio/sdk
+/tmp
```
Comment (SkyKalazar) [inline .gitignore:50]: jhkjh

Comment (AlleyCatRacer) [reply .gitignore:50]: asdf

File: .gitignore (hunk 1 above)
Comment (AlleyCatRacer) [reply .gitignore:39]: Acutely gleaned, good sir!

Comment (AlleyCatRacer) [reply .gitignore:39]: Yes, yes it is.

**Pull request merged**